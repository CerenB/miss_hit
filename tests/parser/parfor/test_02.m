% (c) Copyright 2019 Florian Schanda

function test_02(x)
  parfor (i = 1:x, 8)
    disp(i);
  end
end
