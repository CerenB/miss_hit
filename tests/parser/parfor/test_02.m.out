----------------------------------------------------------------------
--  Parse tree for test_02.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_02>
      Inputs: Identifier <x>
    Body: Sequence_Of_Statements
      Statements: Parallel_For_Statement
        Identifier: Identifier <i>
        Range: Range_Expression
          First: Number_Literal <1>
          Last: Identifier <x>
        Workers: Number_Literal <8>
        Body: Sequence_Of_Statements
          Statements: Naked_Expression_Statement
            Expression: Reference
              Name: Identifier <disp>
              Arguments: Identifier <i>
----------------------------------------------------------------------
In test_02.m, line 3
| function test_02(x)
|                 ^ info: this token is not linked to the ast
In test_02.m, line 3
| function test_02(x)
|                   ^ info: this token is not linked to the ast
In test_02.m, line 4
|   parfor (i = 1:x, 8)
|                ^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
